### DISCUSSION TITLE
Put APK release on Github

### DISCUSSION THREAD

**Discussion by kmcbest** [khoadng/Boorusama#451] [Ideas]
Would be great to have apk download link here, since many devices don't have access to Google Play store.

**Comment by khoadng** [comment]
You can download the app on various alt stores.

**Reply by kmcbest** [reply]
Strangely, all of the stores switched to xapk even though Boorusama's size is less than 100MB. And xapk's are not convenient to install.

**Comment by khoadng** [comment]
Supported in 4.5.0
